src/Tools/Microsoft.VisualStudio.Templates.Editorconfig.Wizard/TemplateConstants.cs

```diff
@@ -15,6 +15,15 @@ internal static class TemplateConstants
 # All files
 [*]
 indent_style = space
+
+# XML project files
+[*.{{csproj,vbproj,vcxproj,vcxproj.filters,proj,projitems,shproj}}]
+indent_size = 2
```

Comment:
Does Visual Studio even honour these? I seem to remember .editorconfig not working in the XML editor in VS.

Having said that, I'm not against this change regardless ¯\\\_(ツ)_/¯

Reply:
> Does Visual Studio even honour these? I seem to remember .editorconfig not working in the XML editor in VS.

I have no idea :/
If it doesn't honor it, I think it will be a good enhancement to support that. Is there any issue tracking that?

Reply:
https://github.com/dotnet/project-system/issues/5427